scenes/Restaurant/RestaurantScene.tscn~RFa3ceedcd.TMP

```diff
@@ -0,0 +1,663 @@
+[gd_scene load_steps=46 format=2]
```

Comment:
What are these "TMP" files? they probably don't belong in git

Reply:
Removed TMP files. Perhaps we should add them to gitignore if they come up again; not sure what generated them.